src/pfe/portal/modules/project/Links.js

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+const fs = require('fs-extra');
+const path = require('path');
+
+const ProjectLinkError = require('../utils/errors/ProjectLinkError');
+const Logger = require('../utils/Logger');
+const log = new Logger(__filename);
+
+const ENV_FILE_NAME = '.codewind-project-links.env'; // This will be in the users container
+
+/**
+ * The Links class
+ */
+class Links {
+  constructor(directory, args) {
+    this._links = (args && args._links) ? args._links : [];
```

Comment:
We discussed whether it would be better index the `links` structure by env var name. This can be changed in a follow up PR if you think it might be easier.

Reply:
I'll change this in a follow up PR as it would clash with next pr.